base/MAPL_SwathGridFactory.F90

```diff
@@ -0,0 +1,1405 @@
+#include "MAPL_Exceptions.h"
+#include "MAPL_ErrLog.h"
+#include "unused_dummy.H"
+
+! Swath: (misisng time and long-axes for scan)
+! This module generates ESMF_Grids for logically-rectangular grid based on observation (netCDF)
+! Spacing between lats (lons) needs not be constant (imhomogenous)
+! LatLon concepts such as corners, pole and dataline are eliminated
+
+module MAPL_SwathGridFactoryMod
+   use MAPL_AbstractGridFactoryMod
+   use MAPL_MinMaxMod
+   use MAPL_KeywordEnforcerMod
+   use MAPL_ExceptionHandling
+   use MAPL_ShmemMod
+   use mapl_ErrorHandlingMod
+   use MAPL_Constants
+   use MAPL_plain_netCDF_Time
+!   use MAPL_FileMetadataUtilsMod
+   use ESMF
+   use pFIO
+   use MAPL_CommsMod
+   use netcdf
+   !!use MAPL_plain_netCDF_Time
+   use, intrinsic :: iso_fortran_env, only: REAL32
+   use, intrinsic :: iso_fortran_env, only: REAL64
+   implicit none
+   integer, parameter :: gridLabel_max = 20
+   private
+
+   public :: SwathGridFactory
+
+!!!   integer, parameter :: NUM_DIM = 2
+   
+   type, extends(AbstractGridFactory) :: SwathGridFactory
+      private
+      character(len=:), allocatable :: grid_name
+      character(len=:), allocatable :: grid_file_name      
+      ! Grid dimensions
+      integer :: im_world = MAPL_UNDEFINED_INTEGER
+      integer :: jm_world = MAPL_UNDEFINED_INTEGER
+      integer :: lm = MAPL_UNDEFINED_INTEGER
+      ! prepare to delete
+      !real(kind=REAL64), allocatable :: lon_centers(:)
+      !real(kind=REAL64), allocatable :: lat_centers(:)
```

Comment:
If we don't need these, then delete.
```suggestion
```

Reply:
deleted together with another unused instanced.